src/main/kotlin/step2/Operator.kt

```diff
@@ -0,0 +1,19 @@
+package step2
+
+enum class Operator(expr: String, evaluate: (num1: Int, num2: Int) -> Int) {
+    PLUS("+", { num1: Int, num2: Int -> num1 + num2 }),
+    MINUS("-", { num1: Int, num2: Int -> num1 - num2 }),
+    MULTIPLY("*", { num1: Int, num2: Int -> num1 * num2 }),
+    DIVIDE("/", { num1: Int, num2: Int -> num1 / num2 });
+
+    var display: String = expr
+        private set
+    var evaluate = evaluate
+        private set
```

Comment:
주 생성자 매개 변수 이름 앞에 `val` 또는 `var`를 추가하여 프로퍼티 정의 및 초기화를 간략히 쓸 수 있습니다.

```suggestion
enum class Operator(val expr: String, val evaluate: (num1: Int, num2: Int) -> Int) {
    PLUS("+", { num1: Int, num2: Int -> num1 + num2 }),
    MINUS("-", { num1: Int, num2: Int -> num1 - num2 }),
    MULTIPLY("*", { num1: Int, num2: Int -> num1 * num2 }),
    DIVIDE("/", { num1: Int, num2: Int -> num1 / num2 });
```

Reply:
@wotjd243 음 `Operator(val expr: String, val evaluate: (num1: Int, num2: Int) -> Int)` 요걸 안쓴 이유가
자동으로 public setter가 생성되어서 필드를 선언하게 되었는데요
말씀하신대로 하면 public setter가 생성되는것 같은데, public setter을 열어두시나요?
아니면 Enum이라서 감안하고 가시는걸까요?

Reply:
요건 재성님과 DM으로 많이 이야기나눈 부분이라 리뷰 반영해두었습니다 :)